node.js

```diff
@@ -53,6 +54,18 @@ function useColors() {
   }
 }
 
+function disableColorDecorations() {
+  var debugColorDecor = (process.env.DEBUG_DISABLE_COLORDECOR || '').trim().toLowerCase();
+  if (0 === debugColorDecor.length) {
+    return false
+  } else {
+    return '0' !== debugColorDecor
```

Comment:
what about `return ! /0|no|false|disabled/.test(debugColorDecor)` ?
Put the regex in a constant and use it in `useColors()` too ?